internal/runbits/commitid/commitid.go

```diff
@@ -0,0 +1,33 @@
+package commitid
```

Comment:
Since this invoked the prompter I think it makes sense you put it in as a runbit, though the naming of the package and method is triggering my OCD a little.

Open to other naming, but how about `commitmediator.Get()`?

Reply:
I had no clue what name to use, so I'm glad for your suggestion.